src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -970,6 +973,11 @@ def customizePerJob(self, job):
         if job.get('inputDatasetLocations'):
             jdl.append('+DESIRED_CMSDataLocations = "%s"\n' % ','.join(job['inputDatasetLocations']))
 
+        # HighIO jobs
+        highIOjob = job.get('highIOjob')
+        if highIOjob:
+            jdl.append('+Requestioslots = 1\n')
```

Comment:
Now admittedly I haven't tried matching a single core highIO job to a single core pilot. It matches fine as a standard single core job to a multi core partitionable pilot without highIO job slots though.